app/models/user.rb

```diff
@@ -146,9 +146,9 @@ def assignable_roles
 
   def logs_filters(specific_org: false)
     if (support? && !specific_org) || organisation.has_managing_agents?
-      %w[status years user organisation]
+      %w[status years user organisation bulk_upload_id]
     else
-      %w[status years user]
+      %w[status years user bulk_upload_id]
```

Comment:
if we have some kind of filter service that wasn't a module we could add this code there. however it seems odd to have to change the user model for filtering which is rather unrelated